metal/eip_controlplane_reconciliation.go

```diff
@@ -148,8 +148,11 @@ func newControlPlaneEndpointManager(k8sclient kubernetes.Interface, stop <-chan
 			},
 		},
 	)
+	if err != nil {
+		return m, err
```

Comment:
I'd argue we could return `nil, err`, but `m, err` is closer to the previous behavior. `m` will presumably be `nil` too.

Reply:
So...we're agreeing to leave it as m, err?